pkg/host-disk/host-disk.go

```diff
@@ -25,14 +25,21 @@ import (
 	"path"
 	"syscall"
 
+	"kubevirt.io/kubevirt/pkg/log"
+
 	k8sv1 "k8s.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/api/v1"
 	"kubevirt.io/kubevirt/pkg/kubecli"
 	"kubevirt.io/kubevirt/pkg/util/types"
 )
 
-const pvcBaseDir = "/var/run/kubevirt-private/vmi-disks"
+const (
+	pvcBaseDir                    = "/var/run/kubevirt-private/vmi-disks"
+	lessPvcSpaceTolerationEnvName = "LESS_PVC_SPACE_TOLERATION"
+	EventReasonToleratedSmallPV   = "ToleratedSmallPV"
+	EventTypeToleratedSmallPV     = "Normal"
```

Comment:
There exists a `k8sv1.EventTypeNormal` constant.

Reply:
thx, using it now